.gitignore

```diff
@@ -21,20 +21,25 @@ lib/galaxy/web/framework/static/scripts
 lib/galaxy/web/framework/static/style
 
 # Database stuff
+database/*.sqlite
 database/beaker_sessions
 database/citations
 database/community_files
 database/compiled_templates
+database/dependencies
 database/files
-database/jobs_directory
 database/job_working_directory
+database/jobs_directory
+database/mulled
+database/object_store_cache
+database/openid_consumer_cache
 database/pbs
 database/shed_tools
 database/test_errors
 database/tmp
-database/*.sqlite
-database/openid_consumer_cache
-database/dependencies
+database/tool_cache
+database/tool_search_index
```

Comment:
Thank you @dannon!
Can we also add `database/objects` ?

Reply:
Sure, what goes in there?

Reply:
I guess, it stores tool output blobs. LIke... Fastqc files are stored there (at least on my system)